cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -441,6 +570,24 @@ Search for properties and tags by specifying one of:
 
 - a complete or partial value, such as ``prod*``; this will return both properties and tags
 
+Since CDAP also annotates *system* metadata to entities by default as mentioned at <link>, the following
+*special* search queries are also supported:
+
+- artifacts or applications containing a specific plugin: ``plugin:<plugin-prefix>*``
+
+- programs with a specific mode: ``batch`` or ``realtime``
+
+- applications with a specific program type: ``flow:<flow-name-prefix>*``, ``service:<service-name-prefix>*``,
+  ``mapreduce:<mapreduce-name-prefix>*``, ``spark:<spark-name-prefix>*``, ``worker:<worker-name-prefix>*``,
+  ``workflow:<workflow-name-prefix>*``
+
+- workflows with the specified node name: ``<workflow-node-name-prefix>*``
+
+- datasets, streams or views with the specified field:
+  - field name only: ``<field-name-prefix>*``
+  - field type only: ``<field-type-prefix>*``
+  - field name with a particular type: ``<field-name-prefix>*:<field-type-prefix>*``
```

Comment:
Also, why is every example in the above section talking about prefix searches? Except for field type searches, everything else can be done using non-prefix search too, right?